src/DynamoCore/BuiltInAndOperators/Operators.xml

```diff
@@ -6,106 +6,119 @@
-                Equal x to y?
+                Compares if x and y are equal
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Basic type object to compare (e.g. bool, int, double, string, etc.)</param>
+            <param name="y">Basic type object to compare (e.g. bool, int, double, string, etc.)</param>
+            <returns name="bool">Result of equality check</returns>
             <search>eq, equal, =</search>
         </member>
 		<member name="%ge">
             <summary>
-                x greater or equals y?
+                Compares if x is greater or equals to y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="bool">True if x is equal or greater than y, false if not</returns>
             <search>greater,equal</search>
         </member>
 		<member name="%gt">
             <summary>
-                x greater y?
+                Compares if x is greater than y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="bool">True if x is greater than y, false if not</returns>
             <search>greater</search>
         </member>
 		<member name="%mod">
             <summary>
-                Finds the remainder of x/y
+               Modulo: Finds the remainder of x/y
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="number">Remainder of the x input after being divided by the y input</returns>
             <search>mod, modulo, remainder</search>
         </member>
 		<member name="%le">
             <summary>
-                x less or equals y?
+              If the x input is less than the y input, or if the two values are equal, it will return true. If x is greater than y, this operator will return false.
             </summary>
-            <param name="x">x value.</param>
-            <param name="y">y value.</param>
+            <param name="x">Integer or double value</param>
+            <param name="y">Integer or double value</param>
+            <returns name="bool">Result of check</returns>
             <search>less,equal</search>
         </member>
 		<member name="%lt">
             <summary>
-                x less y?
+              If the x input is less than the y input, it will return true. If x is greater than y, or if the two value are equal, this operator will return false.
```

Comment:
NIT: My previous comment applies here as well.

Reply:
Addressed in 0e11365